### DISCUSSION TITLE
Floodlight V1 Ethernet possible?

### DISCUSSION THREAD

**Discussion by RequiemFURY** [gtxaspec/wz_mini_hacks#722] [General]
I have two of the classic v1 floodlights with attached v3 cameras. My understanding is the cam attached is just a v3 however I am not sure if there is anyway to get Ethernet to the camera since it uses the pigtail into the floodlight for power and I assume controlling the floodlight but I can’t confirm this. I saw discussion #467 and was able to get RTSP and SSH working fine. In the app, the floodlight is recognized and I can change settings. Ethernet is the only question mark. Has anyone been successful with this?

**Comment by gtxaspec** [comment]
yes, with the following items.

USB Hub with a USB Micro:
https://www.amazon.com/MakerSpot-Accessories-Charging-Extension-Raspberry/dp/B01JL837X8/
USB micro fmale to USB A:
https://www.amazon.com/Pack-Micro-Female-Adapter-Converter/dp/B0986SYF6
Usb Ethernet adapter:
https://www.amazon.com/UGREEN-Ethernet-Adapter-Nintendo-Chromebook/dp/B00MYT481C

**Reply by RequiemFURY** [reply]
Wow, thanks for the quick response and I’ll give this a shot! The link for the USB Micro Female to USB A is not working but I have a few of these guys laying around that I will try with.
https://www.amazon.com/dp/B07SDB7XY1?starsLeft=1&ref_=cm_sw_r_cso_cp_apin_dp_J305FM7BNV6ZJPHQJP95

Hub cable into floodlight base, V3 into the hub using the female Micro to usb-A adapter, Ethernet to hub? 
Is it really that simple?!

**Reply by gtxaspec** [reply]
Hub into the camera, flood light into USB hub,USB Ethernet into hub.
